admin docs

pohly opened the pull request:
I think this is enough to consider #229 resolved. The other two commits are drive-by enhancements.

okartau commented on README.md:
```diff
@@ -555,26 +557,47 @@ These are the steps for manual set-up of certificates:
+
+For each Kubernetes version, four different deployment variants are provided:
+
+-- `direct` or `lvm`: one uses DeviceMode:Direct, the other Device:LVM
```
- Device:LVM should be DeviceMode:LVM
- production should be inside single quotes as well?
- double-dash as startmark does not work as intended, result is two lines do not appear on separate lines (spotted  in preview)

pohly replied:
>    Device:LVM should be DeviceMode:LVM

Fixed.

>    production should be inside single quotes as well?

No, it's not a literal part of the filenames. I wanted to highlight that there are again two variants, but that wasn't obvious. Now it just lists `testing` - is that clearer?

>   double-dash as startmark does not work as intended, result is two lines do not appear on separate lines (spotted in preview)

That was meant to be sub-items. The correct syntax is indented instead of double dashes,  but I am not sure whether that gets rendered as such now - probably not. I kept the indented dashes nevertheless because it reads better in the README.md source.

marcemq commented on README.md:
```diff
@@ -555,26 +557,47 @@ These are the steps for manual set-up of certificates:
+
+For each Kubernetes version, four different deployment variants are provided:
+
+   - `direct` or `lvm`: one uses DeviceMode:Direct, the other DeviceMode:LVM
```
Shouldn't DeviceMode:Direct and DeviceMode:LVM be within quotes? 
i.e.: `DeviceMode:Direct`

marcemq replied:
And yes, a final bullet is missing to this first bullet.

pohly replied:
Added.

Is it okay now?

@okartau if @marcemq doesn't get around to giving her okay, please go ahead and merge. We need to get the number of pending PRs down if we want to release this week.

marcemq replied:
Yes, it is ok now.

marcemq commented on README.md:
```diff
@@ -555,26 +557,47 @@ These are the steps for manual set-up of certificates:
+
+   - `direct` or `lvm`: one uses DeviceMode:Direct, the other DeviceMode:LVM
+   - `testing`: the variants with `testing` in the name enable debugging
+     features and shouldn't be used in production
```
Missing final dot.

marcemq commented on README.md:
```diff
@@ -555,26 +557,47 @@ These are the steps for manual set-up of certificates:
+
+All of these files use `PMEM_REGISTRY` as placeholder for the actual Docker registry
+that contains the PMEM-CSI image. This needs to be replaced as part of the command which
+deploys the to Kubernetes.
```
I think _**deploys the to Kubernetes**_ is missing something.

Is it:
* deploys PMEM-CSI to Kubernetes?

marcemq commented on README.md:
```diff
@@ -555,26 +557,47 @@ These are the steps for manual set-up of certificates:
+possible to customize these variants further with `kustomize`, but one
+has to use a version which supports the `--load_restrictor none` parameter.
+The `Makefile` can be used to build a suitable `kustomize`:
+
```
No need to have two new lines here, we can remove one.

pohly commented:
Thanks for the good catches.

Marcela Morales Q <notifications@github.com> writes:
> Shouldn't DeviceMode:Direct and DeviceMode:LVM be within quotes? 
> i.e.: `DeviceMode:Direct`

Perhaps, but it's not quoted elsewhere either, so let's follow that
usage consistently also here.

> +For each Kubernetes version, four different deployment variants are provided:
> +
> +   - `direct` or `lvm`: one uses DeviceMode:Direct, the other DeviceMode:LVM
> +   - `testing`: the variants with `testing` in the name enable debugging
> +     features and shouldn't be used in production
>
> Missing final dot.

Is another dot then also missing for the first bullet item? I'm not
sure. I've added one dot for now.

> +All of these files use `PMEM_REGISTRY` as placeholder for the actual Docker registry
> +that contains the PMEM-CSI image. This needs to be replaced as part of the command which
> +deploys the to Kubernetes.
>
> I think _**deploys the to Kubernetes**_ is missing something.
>
> Is it:
> * deploys PMEM-CSI to Kubernetes?

"deploys to Kubernetes" is enough, what gets deployed is obvious from
the context.

> +The `Makefile` can be used to build a suitable `kustomize`:
> +
>
> No need to have two new lines here, we can remove one.

Fixed.
